src/main/scala/com/mozilla/telemetry/streaming/ErrorAggregator.scala

```diff
@@ -20,7 +20,7 @@ import org.joda.time.DateTime
 object ErrorAggregator {
 
   private val allowedDocTypes = List("main", "crash")
-  private val outputPrefix = "error_aggregates/v1"
+  private val outputPrefix = "error_aggregates/v2"
```

Comment:
It seems odd to me that starting a new job is required for some types of schema evolution, but not others. It is definitely not ideal that we would have to bump versions and backfill all of the data on dimension changes.

If this is indeed the limitation, one option is we partition by `submission_date`, start running the new schema, and then copy over the data from the previous `submission_dates`. We can either backfill the overlap day for the new schema, or simply copy over the old schema's data for that day. Then we don't need to backfill, just copy the files on schema evolution compatible schema changes.